airbyte-integrations/connectors/source-pendo-python/source_pendo_python/schemas/report_result.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "object",
+  "$schema": "http://json-schema.org/schema#",
+  "properties": {
+    "reportId": {
+      "type": "string"
+    },
+    "visitorId": {
+      "type": "string"
+    },
+    "daysActive": {
+      "type": "integer"
+    },
+    "eventCount": {
+      "type": "integer"
+    },
+    "usageTrending": {
+      "type": "integer"
+    },
+    "visitor_auto_lastvisit": {
+      "type": "integer"
+    }
```

Comment:
My understanding is that the report results have a variable schema; I think we'll need to do some amount of inspection to dynamically generate the schema.  (e.g. build from first row of results)